arches/app/etl_modules/save.py

```diff
@@ -11,16 +11,16 @@
 logger = logging.getLogger(__name__)
 
 
-def save_to_tiles(userid, loadid):
+def save_to_tiles(userid, loadid, multiprocessing=False):
     with connection.cursor() as cursor:
         disable_tile_triggers(cursor, loadid)
         error_saving_tiles = _save_to_tiles(cursor, loadid)
         reenable_tile_triggers(cursor, loadid)
         if error_saving_tiles:
             return error_saving_tiles
 
-        _post_save_edit_log(cursor, userid, loadid)
-
+        response = _post_save_edit_log(cursor, userid, loadid, multiprocessing)
+    return response
```

Comment:
Thanks @njkim for catching this -- can you also use this opportunity to fix JSONLDImporter.save_to_tiles to also return the result of _post_save_edit_log()?